test/AngularSpec.js

```diff
@@ -1671,6 +1671,110 @@ describe('angular', function() {
     });
   });
 
+  describe('shutdown', function() {
```

Comment:
We should test that event handlers are removed as well (current code doesn't do that when jQuery is used).